packages/api-v2/src/auth/auth.controller.ts

```diff
@@ -17,7 +19,7 @@ export class AuthController {
   @Post("register")
   public async register(
     @Body() createStudentDto: CreateStudentDto
-  ): Promise<Student> {
+  ): Promise<GetStudentResponse> {
```

Comment:
thoughts on changing the name back to student?

Reply:
hmmm... GetStudentResponse makes more sense to me since it decouples the response from the resource, but i'm ok with either